romana-install/group_vars/vagrant

```diff
@@ -1,9 +1,11 @@
 ram_size:
   controller:
     devstack: 4096
+    openstack: 4096
     kubernetes: 2048
   compute:
     devstack: 1024
+    openstack: 1024
```

Comment:
Weird that kubernetes nodes get more memory than openstack and devstack nodes, right?